### DISCUSSION TITLE
producer 延迟故障机制

### DISCUSSION THREAD

**Discussion by LittleBoy18** [apache/rocketmq#7897] [Q&A] [answered]
rocketmq 5.x proxy是否还支持producer的延迟故障机制？

**Comment by LittleBoy18** [comment]
4.x可以通过producer.setSendLatencyFaultEnable(true)开启

**Comment by GenerousMan** [comment, marked as answer]
Yes, this has already been supported in the 5.0 proxy, introduced through RIP-63. There are relevant configurations in ProxyConfig: startDetectorEnable/startDetectorEnable.
